src/app/shared/services/shared.service.ts

```diff
@@ -56,10 +56,48 @@ export class SharedService {
 
   }
 
+  /**
+   * Gets arbol cuenta contable
+   *  returns one tree level at once.
+   * @param [branch] tree's branch to request info from the API
+   * @returns  branch information.
+   */
+   public getArbolCuentaContableDebito() {
+    this.rqManager.setPath('CUENTAS_CONTABLES_SERVICE');
+    // this.rqManager.setPath('DUMMY_SERVICE');
+    // Set the optional branch for the API request.
+    // const unidadEjecutora = 1;
+    // call request manager for the tree's data.
+    return this.rqManager.get(`nodo_cuenta_contable/cuentas/debito`);
+
+  }
+
+  /**
+   * Gets arbol cuenta contable
+   *  returns one tree level at once.
+   * @param [branch] tree's branch to request info from the API
+   * @returns  branch information.
+   */
+   public getArbolCuentaContableCredito() {
+    this.rqManager.setPath('CUENTAS_CONTABLES_SERVICE');
+    // this.rqManager.setPath('DUMMY_SERVICE');
+    // Set the optional branch for the API request.
+    // const unidadEjecutora = 1;
+    // call request manager for the tree's data.
```

Comment:
```suggestion
    // Set the optional branch for the API request.
    // call request manager for the tree's data.
```
O confirmar si se requieren estos comentarios...